templates/nextflow-s3-template.yaml

```diff
@@ -0,0 +1,177 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Private S3 Bucket for Nextflow
+
+# Parameters
+Parameters:
+  Department:
+    Description: 'The department for this resource'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  Project:
+    Description: 'The name of the project that this resource is used for'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  OwnerEmail:
+    Description: 'Email address of the owner of this resource'
+    Type: String
+    AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
+    ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
```

Comment:
These parameters aren't being used. We can remove them and rely on `stack_tags` instead ([example](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/blob/main/config/prod/nextflow-aurora-mysql.yaml#L10-L13)), which means you just need to delete these lines since `stack_tags` are specified in the config when the template is deployed. You can just accept my suggested edit below to do this quickly on GitHub. 

```suggestion
```